lib/GridItem.jsx

```diff
@@ -1,18 +1,16 @@
 // @flow
-import {default as React, Component} from 'react';
-import {perc, setTransform} from './utils';
+import React from 'react';
 import Draggable from 'react-draggable';
 import {Resizable} from 'react-resizable';
-import assign from 'object-assign';
+import {perc, setTransform} from './utils';
 
-import type {Position, Size} from './utils';
+import type {Position} from './utils';
 
 /**
  * An individual item within a ReactGridLayout.
  */
-export default class GridItem extends Component {
-
-  static propTypes = {
+const GridItem = React.createClass({
```

Comment:
Understood. I rather like ES6 classes and I see the majority of new React libs being built in them. The effect is the same. It's useful with Flow as well for easily e.g. typechecking State.

It doesn't matter much - I may revert the createClass switch but the meat of this PR doesn't depend on that. Thanks.